innsending/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/innsending/UtledBestemmendeFravaersdag.kt

```diff
@@ -0,0 +1,22 @@
+package no.nav.helsearbeidsgiver.inntektsmelding.innsending
+
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.Inntektsmelding
+import no.nav.helsearbeidsgiver.domene.inntektsmelding.v1.bestemmendeFravaersdag
+import no.nav.helsearbeidsgiver.felles.domene.Forespoersel
+import java.time.LocalDate
+
+fun utledBestemmendeFravaersdag(
+    forespoersel: Forespoersel,
+    inntektsmelding: Inntektsmelding,
+): LocalDate =
+    if (
+        forespoersel.forespurtData.arbeidsgiverperiode.paakrevd ||
+        (!forespoersel.forespurtData.inntekt.paakrevd && forespoersel.forespurtData.refusjon.paakrevd)
```

Comment:
Er det mulig å få en kort domene/regel-forklaring på bakgrunnen for at logikken er sånn den er her? 🙏 Det ser ut som det at refusjon er påkrevd eller ikke har noe å si for hvilken bestemmende fraværsdag vi ender opp med?

Reply:
Bedre nå?

Reply:
![pacha-okay](https://github.com/user-attachments/assets/6cf7bd68-9020-4c04-98bf-a0e5fe458cb6)

Reply:
Kongerike for en lama 😍